topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: tutorial_hands_on
+topic_name: transcriptomics
+tutorial_name: rna-seq-viz-with-cummerbund
+---
+
+# Introduction
+{:.no_toc}
+
+RNA-Seq analysis helps researchers finding new genes and spliced variants, and provides a quantification of cell- and context- specific gene expression. Its data is however complex to handle, and requires both computer science and mathematical knowledge to be managed and interpreted.
+
+Visualization techniques are key to overcome the complexity of RNA-Seq data, and represent valuable tools to gather information and insights.
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Reasons for visualizing RNA-Seq results
+
+To make sense of the available RNA-Seq data, and overview the condition-specific gene expression levels of the provided samples, we need to visualize our results. Here we will use [CummeRbund](http://compbio.mit.edu/cummeRbund/).
+
+CummeRbund is an open-source tool that simplifies the analysis of a CuffDiff RNA-Seq output. In particular, it helps researchers:
+- managing, integrating, and visualizing the data produced by CuffDiff
+- simplifying data exploration
+- providing a bird's-eye view of the expression analysis by describing relationships betweeen genes, transcripts, transcription start sites, and CDS regions
+- exploring subfeatures of individual genes or gene-sets
+- creating publication-ready plots
+
+A typical workflow for the visualization of RNA-Seq data involving CummeRbund:
+
+![workflow](../../images/cummerbund-rna-seq-viz-with-cummerbund.png)
+
+CummeRbund reads your RNA-Seq results from a [SQLite](https://www.sqlite.org/) database. This database has to be created using CuffDiff's SQLite output option.
+
+> ### {% icon tip %} Tip: SQLite output with CuffDiff
+>
+> Instruct CuffDiff to organize its output in a SQLite database for later read with CummeRbund
```

Comment:
Minor grammar change:

Instruct CuffDiff to organize its output in a SQLite database to be read by CummeRbund.